bombastic/index/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 bombastic-model = { path = "../model" }
-cyclonedx-bom = "0.4.0"
+cyclonedx-bom = { git = "https://github.com/CycloneDX/cyclonedx-rust-cargo", rev = "0aa9ee0c9b28d6b92772330baea957d67f500670" }
```

Comment:
It makes more sense to use the `patch` section of the workspace level Cargo.toml  file.